### DISCUSSION TITLE
Noob question: What are the mass storage options available in the pack?

### DISCUSSION THREAD

**Discussion by LumosX** [NillerMedDild/Enigmatica6#3322] [Q&A] [answered]
After an itch to play some modded minecraft, I've set up a server using E6 (months ago, really, but I only play a bit at a time). Great modpack so far, really enjoying everything though I'm a complete noob. I've not really played any modded ever, nor have I watched any minecraft youtube (nor played vanilla MC) since 2015, so I'm a little out of the loop regarding everything, and the first "issue" I'm really running into is storage. The questbook lists a plethora of storage options, but none of them seem match what I'm looking for.

In short, what options are there for mass storage? I want to dump everything in one gargantuan inventory and then be able to access items as I please. I've seen certus quartz ore, so I gather Applied Energistics is available; but considering I've just found my first diamonds, are there any earlier-game alternatives for unified mass storage?

(In truth, I did try out some 1.12 modpack a while back (was far too laggy though), and added ProjectE on top of it. Dumping everything into a transmutation table really makes one's life much easier, but I want to keep the E6 experience unblemished, and am resisting the temptation to just install ProjectE again.)

Do excuse me if an answer was readily available somewhere else, or in the questbook, but I've not heard of most of the mods in this pack, let alone used them, so help will be appreciated. :)

**Comment by TemperedFool** [comment, marked as answer]
The best place to ask questions like this would be the discord, in #enigmatica6: https://discord.gg/enigmatica

The quest book should list a number of options; to name 2:

AE2 is not part of the pack, but Refined Storage is and is the general answer to this. Very large discs are available, and without types you can store a lot. Just beware dumping tons of unique items with NBT (say, enchanted items from a mob farm), as it is possible to jam things up if you're too careless.

The other big mass-storage option is Occultism - it has its own extremely large storage system, complete with cross-dimensional wireless access and no maintenance costs.

**Reply by LumosX** [reply]
Thanks, I appreciate it! Guess I'll look into Occultism.
